doc/source/user_guide/numpy_images.txt

```diff
@@ -55,28 +64,134 @@ Masking (indexing with masks of booleans)::
 
 Fancy indexing (indexing with sets of indices) ::
 
-    >>> inds_x = np.arange(len(camera))
-    >>> inds_y = 4 * inds_x % len(camera)
-    >>> camera[inds_x, inds_y] = 0
+    >>> inds_r = np.arange(len(camera))
+    >>> inds_c = 4 * inds_r % len(camera)
+    >>> camera[inds_r, inds_c] = 0
 
 Using masks, especially, is very useful to select a set of pixels on
 which to perform further manipulations. The mask can be any boolean array
-of same shape as the image (or at least a shape broadcastable to the
-image shape). This can be useful to define a region of interest, as a
+of same shape as the image (or a shape broadcastable to the image shape).
+This can be useful to define a region of interest, such as a
 disk: ::
 
-    >>> l_x, l_y = camera.shape[0], camera.shape[1]
-    >>> X, Y = np.ogrid[:l_x, :l_y]
-    >>> outer_disk_mask = (X - l_x / 2)**2 + (Y - l_y / 2)**2 < (l_x / 2)**2
+    >>> nrows, ncols = camera.shape
+    >>> row, col = np.ogrid[:nrows, :ncols]
+    >>> cnt_row, cnt_col = nrows / 2, ncols / 2
+    >>> outer_disk_mask = ((row - cnt_row)**2 + (col - cnt_col)**2 <
+    ...                    (nrows / 2)**2)
     >>> camera[outer_disk_mask] = 0
 
 .. image:: ../../_images/plot_camera_numpy_1.png
     :width: 45%
     :target: ../auto_examples/plot_camera_numpy.html
 
-Boolean arithmetics can be used to define more complex masks: ::
+Boolean arithmetic can be used to define more complex masks: ::
 
-    >>> lower_half = X > l_x / 2
+    >>> lower_half = row > cnt_row
     >>> lower_half_disk = np.logical_and(lower_half, outer_disk_mask)
     >>> camera = data.camera()
     >>> camera[lower_half_disk] = 0 
+
+
+Color images
+------------
+
+All of the above is true of color images, too: a color image is a
+NumPy array, with an additional trailing dimension for the channels:
+
+    >>> cat = data.chelsea()
+    >>> type(cat)
+    <type 'numpy.ndarray'>
+    >>> cat.shape
+    (300, 451, 3)
+
+That's a 300-by-451 pixel image with red, green, and blue channels.
```

Comment:
that is ? don't know if we should stick to a very academic style in the documentation :-)